apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -186,15 +189,23 @@ def render(element_html: str, data: pl.QuestionData) -> str:
 
         if parse_error is None:
             a_submitted = pl.from_json(data["submitted_answers"].get(name, None))
-            if a_submitted is not None and len(a_submitted.shape) == 2:
+            if (
+                a_submitted is not None
+                and type(a_submitted) is np.ndarray
```

Comment:
Shouldn't this be done with isinstance?

Reply:
Changed in c16f47fdd511d607361168508a27a884a2d02121.